[1.37.0] Release notes (#447)
replicatedhq/kots.io: content/release-notes/1.37.0.md

---
date: 2021-04-02T16:30:14
linktitle: "1.37.0"
title: "1.37.0"
kubernetes: "1.18, 1.19, and 1.20"
---

{{<changes>}}
* Updated snapshot [restore CLI](/kots-cli/restore/) to report errors and warnings from Velero.

> * Update UI to include timezone when displaying app version deployment time.

Reviewer: ```suggestion
* Updated UI to include timezone when displaying app version deployment time.
```

* Updated [GitOps integration](/kotsadm/gitops/single-app-workflows/) to use default branch when no branch is specified.
* Updated Postgres to use Alpine as base image instead of Debian Stretch.

> * Updated Schemahero to use Debian Buster as base instead of Stretch.

Reviewer: ```suggestion
* Updated Schemahero to use Debian Buster as base image instead of Debian Stretch.
```

{{</changes>}}

{{<fixes>}}

> * Fixed a bug that could cause [instance restore](/kotsadm/snapshots/disaster-recovery/#airgapped-embedded-cluster-restore) to be be stuck in `In Progress` state indefinitely.

Reviewer: ```suggestion
* Fixed a bug that could cause [full snapshot restore](/kotsadm/snapshots/disaster-recovery/#airgapped-embedded-cluster-restore) to be be stuck in `In Progress` state indefinitely.
```
